app/routes/search/$text.tsx

```diff
@@ -8,7 +8,17 @@ import { getEntriesByBasicTextSearch } from "~/models/entry.server"
 export async function loader({ request, params }: LoaderArgs) {
   invariant(params.text, "text not found")
 
-  const entries = await getEntriesByBasicTextSearch(params.text)
+  const url = new URL(request.url)
+  const caseSensitive: boolean =
```

Comment:
Just making the comment — you don't have to make the change to the code — but we can still probably get away with implicit typing here, e.g. `const caseSensitive = ...` rather than specifying the `boolean`. Minor thing.